.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -5,40 +5,46 @@ contact_links:
     about: "Please use #nds-help on the Discord server to for support-related questions, such as needing assistance to launch TWiLight Menu++"
   - name: DS game issue
     url: https://github.com/DS-Homebrew/nds-bootstrap/issues/new
-    about: Please use the nds-bootstrap GitHub repository to report issues related to running `.nds`, `.dsi`, `.srl`, or `.app` files via nds-bootstrap (and not via Unlaunch, flashcard kernel or a modern homebrew without a RAM disk)
+    about: Please use the nds-bootstrap GitHub repository to report issues related to running .nds, .dsi, .srl, or .app files via nds-bootstrap (and not via Unlaunch, flashcard kernel or a modern homebrew without a RAM disk)
```

Comment:
if we can't use codeblocks, maybe just stick to plain ol' quotes? This is just a personal opinion really but imo file extensions should stand out in some form